tools/examples/src/generate-examples-lib-barrel.ts

```diff
@@ -35,27 +27,55 @@ export async function generateExamplesLibBarrelFile(
 
   return transformAndWriteTemplate(
     source => {
-      let exports: string[] = [];
+      let imports: string[] = [];
```

Comment:
array should be const you don't assign a new array you only push new items to it